cdc/entry/mounter.go

```diff
@@ -207,6 +207,14 @@ func fetchHandleValue(tableInfo *schema.TableInfo, recordID int64) (pkCoID int64
 
 func (m *Mounter) mountDDL(jobEntry *ddlJobKVEntry) (*model.DDL, error) {
 	databaseName := jobEntry.Job.SchemaName
+	// schema name in raw ddl job entry is always lowercase, try to find the
+	// correct value from schema storage
+	if m.schemaStorage != nil {
```

Comment:
we can find origin schema name in `job.BinlogInfo.DBInfo.Name.O`

Reply:
I'm not sure, but from TiDB DDL history the `job.BinlogInfo.DBInfo` is nil

<img width="763" alt="Screen Shot 2020-02-27 at 19 11 21" src="https://user-images.githubusercontent.com/1527315/75439616-17a65f80-5995-11ea-9b1e-6eb5e66c6d4e.png">